src/lib/leaflet-tilelayer-subpixel-fix.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable object-shorthand */
+/* eslint-disable func-names */
```

Comment:
Hey @tienhoah you should probably disable these two rules, `object-shorthand` and `func-names` globally.